### ISSUE TITLE
Installation folder on Windows is missing "alpha" in its name

### ISSUE THREAD

**Issue by HemantMahawar** [PowerShell/PowerShell#2661]
Steps to reproduce
------------------
* Install latest PSCore (alpha.12) on Windows 10 from [releases tab](https://github.com/PowerShell/PowerShell/releases)
* Check the installation folder

Expected behavior
-----------------
It creates a folder 6.0.0-alpha.12 under windir\program files\powershell

Actual behavior
---------------
It creates 6.0.0.12

```powershell
dir 'C:\Program Files\PowerShell\'

    Directory: C:\Program Files\PowerShell


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----       11/10/2016   2:18 PM                6.0.0.12
                                                 ^^^^^^^^
```

**Comment by mirichmo** [comment]
@raghushantha Is this due to a limitation in WiX (or MSI)'s ability to handle version numbers?

**Comment by raghushantha** [comment]
Fixed by #3435
